hack/generator/pkg/astmodel/errored_type.go

```diff
@@ -81,15 +109,27 @@ func (errored ErroredType) handleWarningsAndErrors() {
 
 func (errored ErroredType) AsDeclarations(cgc *CodeGenerationContext, dc DeclarationContext) []ast.Decl {
 	errored.handleWarningsAndErrors()
-	return errored.inner.AsDeclarations(cgc, dc)
+	if errored.inner != nil {
```

Comment:
I think I see one of the reasons you were wishing for interface receivers the other day! I don't think the workaround I was going to suggest of wrapping inner in a struct with nil-checking Type methods would buy much here though. :(

Reply:
@babbageclunk I actually wanted it for the `IgnoringErrors` function 🙂